### ISSUE TITLE
skaffold v1.17.1 installed from `brew` reports wrong version number.

### ISSUE THREAD

**Issue by tejal29** [GoogleContainerTools/skaffold#5104]
@MarlonGamez brought this to my attention. 

Marlon, was checking out the latest release to verify a feature. He noticed, the version number is wrong.`v1.7.1` vs `v1.17.1`

```
brew upgrade
skaffold version
v1.7.1
``` 

This could have been due to a manual error during release. 
When doing the release, @tejal29 had mistakenly create a [release tag](https://docs.google.com/document/d/1wZQqP7If80Sv9_0WvqCtVLmLF2dqdD_t7V7LhyP_gtE/edit#heading=h.yyubpp6lgxps) as `v1.7.1`
see https://github.com/GoogleContainerTools/skaffold/pull/5092 the branch name here.

However, she quickly realized and created another release tag v1.17.1 and deleted the v1.7.1
However, pushing a tag runs the gcloud trigger and it could have built the new binaries with wrong version.

@tejal29 verified, the binaries uploaded to `gcloud` have the correct version
```
$cp ./third_party/cloudsdk/component_build/cloud_tools/skaffold_linux_amd64-1.17.1.zip
 ~/tmp

$cd ~/tmp
$unzip skaffold_linux_amd64-1.17.1.zip
$chmod 755 skaffold
$ ./skaffold version
v1.17.1

```

**Comment by tejal29** [comment]
Steps to update brew
```
Brew update:
brew's bot eventually picks it up when you tag a new release
eg. https://github.com/Homebrew/homebrew-core/commit/a518585b44104ecdcd5138665c5e0aa13cc3f83a but you can run brew bump-formula-pr --help to do it manually

``

**Comment by tejal29** [comment]
Also verified Cloud SDK skaffold mac binary. The version reported it correct.

**Comment by tejal29** [comment]
https://github.com/Homebrew/homebrew-core/pull/66180

**Comment by tejal29** [comment]
Shd be closed by v1.17.2 release.
https://github.com/GoogleContainerTools/skaffold/releases/tag/v1.17.2
